Update C* recovery process (#206)
scalar-labs/scalar-terraform: docs/CassandraOperation.md

sudo mount -a
```

### Restart Cassandra

> As a last step of Cassandra node replacement, you need to configure Cassandra appropriately before start it. What needs to be done depends on the following cases.

Author: ```suggestion
As the last step of Cassandra node replacement, you need to configure Cassandra appropriately before starting it. What needs to be done depends on the following cases.
```

> * Case1: Node (Instance/VM) is repalced (but the volume is not replaced)

Author: ```suggestion
* Case1: Node (Instance/VM) is replaced (but the volume is not replaced)
```

> * Case2: Volume is repalced (but the node is not replaced)

Author: ```suggestion
* Case2: Volume is replaced (but the node is not replaced)
```

* Case3: Node and the volume are both replaced

Here are the steps for each case.

> #### Case1: Node (Instance/VM) is repalced (but the volume is not replaced)

Author: ```suggestion
#### Case1: Node (Instance/VM) is replaced (but the volume is not replaced)
```

* Remove the IP of the new node from seeds in `casssandra.yaml`
* Add `JVM_OPTS="$JVM_OPTS -Dcassandra.replace_address_first_boot=<dead node ip>"` to the bottom of `cassandra-env.sh`
* `sudo systemctl start cassandra`
* Repair with Reaper at some later point 

> #### Case2: Volume is repalced (but the node is not replaced)

Author: ```suggestion
#### Case2: Volume is replaced (but the node is not replaced)
```

* Restore data from Cassy
* `sudo systemctl start cassandra`
* Repair with Reaper at some later point 

#### Case3: Node and the volume are both replaced
* Restore data from Cassy

>   * You need to manually update the IP of an entry of `backup_history` table to match with the new IP

Author: ```suggestion
  * You need to manually update the IP of the entry of `backup_history` table to match with the new IP
```

* Remove the IP of the new node from seeds in `casssandra.yaml`
* Set `auto_bootstrap` to `false` in `cassandra.yaml`
    * this might not be needed. need to verify.
* Add `JVM_OPTS="$JVM_OPTS -Dcassandra.replace_address_first_boot=<dead node ip>"` to the bottom of `cassandra-env.sh`

> * sudo systemctl start cassandra

Author: ```suggestion
* `sudo systemctl start cassandra`
```
